### ISSUE TITLE
Tab context not transferred to image upload dialog

### ISSUE THREAD

**Issue by ArtOfCode-** [codidact/qpixel#1611]
meta:293904

When uploading an image via the toolbar button, tab context is not transferred to the modal, resulting in the Tab key cycling elements on the background page rather than the modal.

**Comment by superplane39** [comment]
The dialog also doesn't close when you press the <kbd>esc</kbd> key, which is also an issue and should be fixed at the same time if possible.

**Comment by cellio** [comment]
New meta post about `esc`: https://meta.codidact.com/posts/293908

**Comment by Oaphi** [comment]
Note to self: <kbd>Esc</kbd> is supposed to be working, we are just using the `keypress` deprecated event.
